frontend/src/components/object/ObjectTable.vue

```diff
@@ -251,9 +252,9 @@ const filters = ref({
               data.public ||
               permissionStore.isObjectActionAllowed(data.id, getUserId, Permissions.READ, props.bucketId as string)
             "
-            v-tooltip.bottom="'Bucket detail'"
+            v-tooltip.bottom="'Bucket details'"
             class="p-button-lg p-button-rounded p-button-text"
-            aria-label="Bucket detail"
+            aria-label="Bucket details"
```

Comment:
Still says Bucket instead of Object

Reply:
Ya, I see some inconsistency as well. Do you think it should be "file details" instead. This will make it consistent with the popup?